qualtricssurvey/models.py

```diff
@@ -4,14 +4,67 @@
 
 import six
 from datetime import datetime
-
+from xblock.scorable import ScorableXBlockMixin, Score
 from django.utils.translation import ugettext_lazy as _
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from xblock.core import XBlock
 from xblock.fields import Scope
-from xblock.fields import Boolean, List, String
-
+from xblock.fields import Boolean, List, String, Float
 from opaque_keys.edx.keys import UsageKey
 from xmodule.modulestore.django import modulestore
+from .mixins.handlers import QualtricsHandlersMixin
+import requests
+import json
+from collections import namedtuple
+from common.djangoapps.student.models import user_by_anonymous_id
```

Comment:
@bsande6 When I ran migrations without having `INSTALLED_APPS.append('qualtricssurvey')` defined in `edx-platform/lms/envs/private.py` I received the following error. Did you run into this issue?

```
2021-04-23 16:40:38,075 WARNING 398 [py.warnings] [user None] __init__.py:142 - /edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/sorl/thumbnail/conf/__init__.py:16: RemovedInDjango31Warning: The FILE_CHARSET setting is deprecated. Starting with Django 3.1, all files read from disk must be UTF-8 encoded.
  setattr(self, attr, getattr(obj, attr))

Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/manage.py", line 120, in <module>
    startup.run()
  File "/edx/app/edxapp/edx-platform/lms/startup.py", line 20, in run
    django.setup()
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/edx/src/xblocks/xblock-qualtrics-survey/qualtricssurvey/models.py", line 19, in <module>
    from common.djangoapps.student.models import user_by_anonymous_id
  File "/edx/app/edxapp/edx-platform/common/djangoapps/student/models.py", line 130, in <module>
    class AnonymousUserId(models.Model):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.5/site-packages/django/db/models/base.py", line 111, in __new__
    "INSTALLED_APPS." % (module, name)
RuntimeError: Model class common.djangoapps.student.models.AnonymousUserId doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

Reply:
Looks like imports are handled different between the releases.

See this for details.
https://github.com/open-craft/problem-builder/blob/master/problem_builder/platform_dependencies.py#L48-L49

Reply:
I updated this with https://github.com/CUCWD/xblock-qualtrics-survey/pull/6/commits/c1f00430b50ab3727a50fd333455c9500bc92156

Reply:
Handled update with https://github.com/CUCWD/xblock-qualtrics-survey/commit/6c1017df7b3e50f120bc7829cc32abf84adf05a0